test/integration/auth/node_test.go

```diff
@@ -275,134 +274,156 @@ func TestNodeAuthorizer(t *testing.T) {
 	node2Client := clientsetForToken(tokenNode2, clientConfig)
 
 	// all node requests from node1 and unknown node fail
-	expectForbidden(t, getSecret(nodeanonClient))
-	expectForbidden(t, getPVSecret(nodeanonClient))
-	expectForbidden(t, getConfigMap(nodeanonClient))
-	expectForbidden(t, getPVC(nodeanonClient))
-	expectForbidden(t, getPV(nodeanonClient))
-	expectForbidden(t, createNode2NormalPod(nodeanonClient))
-	expectForbidden(t, createNode2MirrorPod(nodeanonClient))
-	expectForbidden(t, deleteNode2NormalPod(nodeanonClient))
-	expectForbidden(t, deleteNode2MirrorPod(nodeanonClient))
-	expectForbidden(t, createNode2MirrorPodEviction(nodeanonClient))
-	expectForbidden(t, createNode2(nodeanonClient))
-	expectForbidden(t, updateNode2Status(nodeanonClient))
-	expectForbidden(t, deleteNode2(nodeanonClient))
-
-	expectForbidden(t, getSecret(node1Client))
-	expectForbidden(t, getPVSecret(node1Client))
-	expectForbidden(t, getConfigMap(node1Client))
-	expectForbidden(t, getPVC(node1Client))
-	expectForbidden(t, getPV(node1Client))
-	expectForbidden(t, createNode2NormalPod(nodeanonClient))
-	expectForbidden(t, createNode2MirrorPod(node1Client))
-	expectNotFound(t, deleteNode2MirrorPod(node1Client))
-	expectNotFound(t, createNode2MirrorPodEviction(node1Client))
-	expectForbidden(t, createNode2(node1Client))
-	expectForbidden(t, updateNode2Status(node1Client))
-	expectForbidden(t, deleteNode2(node1Client))
+	expectForbidden(t, getSecret, nodeanonClient)
```

Comment:
I was actually thinking we'd change the operation functions to return `func() error`, rather than plumbing multiple vars through all the layers

Reply:
Original change did that, it was just a much bigger diff.

```go
expectForbidden(t, func() error{ return getSecret(nodeanonClient) })
```

Reply:
Woop, I see what you're saying. Will do that fix.